boards/arm/olimex_stm32_p405/doc/olimex_stm32_p405.rst

```diff
@@ -0,0 +1,270 @@
+
+Pin Mapping
+===========
+
+.. figure:: img/olimex-stm32-p405-front.png
+     :width: 739px
+     :align: center
+     :height: 530px
+     :alt: OLIMEX-STM32-P405 connectors
+
+     OLIMEX-STM32-P405 connectors
+
+LED
+---
+
+* USER_LED (red) = PC12
+* PWR_LED (red) = power
+
+Push buttons
+------------
+
+* USER_BUTTON = PA0
+* RST = NRST
+
+External Connectors
+-------------------
+
+JTAG debug
+
++-------+--------------+-------+--------------+
+| PIN # | Signal Name  | Pin # | Signal Name  |
++=======+==============+=======+==============+
+| 1     | +3.3V        | 11    | -            |
++-------+--------------+-------+--------------+
+| 2     | +3.3V        | 12    | GND          |
++-------+--------------+-------+--------------+
+| 3     | PB4 / TRST   | 13    | PB3 / TDO    |
++-------+--------------+-------+--------------+
+| 4     | GND          | 14    | GND          |
++-------+--------------+-------+--------------+
+| 5     | PA15 / TDI   | 15    | PB4 / TRST   |
++-------+--------------+-------+--------------+
+| 6     | GND          | 16    | GND          |
++-------+--------------+-------+--------------+
+| 7     | PA13 / TMS   | 17    | -            |
++-------+--------------+-------+--------------+
+| 8     | GND          | 18    | GND          |
++-------+--------------+-------+--------------+
+| 9     | PA14 / TCK   | 19    | +5V_JTAG     |
++-------+--------------+-------+--------------+
+| 10    | GND          | 20    | GND          |
++-------+--------------+-------+--------------+
+
+UEXT
+
++-------+-----------------------+----------------+
+| PIN # | Wire   Name           | STM32F405 port |
++=======+=======================+================+
+| 1     | +3.3V                 | -              |
++-------+-----------------------+----------------+
+| 2     | GND                   | -              |
++-------+-----------------------+----------------+
+| 3     | PA9/USART1_TX         | PA9            |
++-------+-----------------------+----------------+
+| 4     | PA10/USART1_RX        | PA10           |
++-------+-----------------------+----------------+
+| 5     | PB6/I2C1_SCL          | PB6            |
++-------+-----------------------+----------------+
+| 6     | PB7/I2C1_SDA          | PB7            |
++-------+-----------------------+----------------+
+| 7     | PA6/SPI1_MISO         | PA6            |
++-------+-----------------------+----------------+
+| 8     | PA7/SPI1_MOSI         | PA7            |
++-------+-----------------------+----------------+
+| 9     | PA5/SPI1_SCK          | PA5            |
++-------+-----------------------+----------------+
+| 10    | PA4/SPI1_NSS          | PA4            |
++-------+-----------------------+----------------+
+
+
+GPIO row of pins
+
++---------+----------------------------------------------+
+| Pin     | STM32F405 Pin Functions                      |
++=======+================================================+
```

Comment:
need to have the `+` column indicator on the heading/row line, align with the previous `|`